### DISCUSSION TITLE
Не работают войсы дискорда

### DISCUSSION THREAD

**Discussion by Sofutikku** [bol-van/zapret#1012] [Q&A]
пытался делать конфиг через blockcheck.sh
[config.txt](https://github.com/user-attachments/files/18276775/config.txt)

**Comment by ghost** [comment]


**Reply by Sofutikku** [reply]
Я дегенерат в этом плане, если можете указать что именно надо читать может пойму

**Reply by Sofutikku** [reply]
я пытался читать все но не смог нормально настроить

**Reply by 8hrsk** [reply]
Хостлист не обязателен. Скорее всего, проблема в том, что у человека неверно составлен конфиг.

@Yawarakaii киньте свой конфиг, чтобы вообще можно было понять, в чем суть проблемы

**Reply by 8hrsk** [reply]
У меня наоборот с хостлистом не заводилось. Перебрал дурилки, настроил udp. Всё работает, блокнутые сайты (Linkedin, animego, discord, yourube) работают на ура. Войсы в дискоре тоже. Использовал автохостлист, да udp без хостлиста

**Reply by Sofutikku** [reply]
> > Хостлист не обязателен. Скорее всего, проблема в том, что у человека неверно составлен конфиг.
> > @Yawarakaii киньте свой конфиг, чтобы вообще можно было понять, в чем суть проблемы
> 
> у меня была такая проблема, я фиксил лично хостлистом. ну да, возможно с конфигом у юзера проблемы.

возможно и вправду хостлист

**Comment by 8hrsk** [comment]
Посмотрел я ваш конфиг. У вас все методы дурения просто выключены 🤣

**Reply by Sofutikku** [reply]
это не настроенный конфиг с blockcheck.sh, я пытался добовлять md5sig но он все равно не помогало

**Reply by 8hrsk** [reply]
Киньте конфиг, который вы пытаетесь использовать. Или это он и есть?

**Reply by Sofutikku** [reply]
> Киньте конфиг, который вы пытаетесь использовать. Или это он и есть?


[config.default.txt](https://github.com/user-attachments/files/18285655/config.default.txt)
конфиг который я использую щас

**Reply by 8hrsk** [reply]
У вас выключены все дурилки в конфиге

**Reply by Sofutikku** [reply]
> У вас выключены все дурилки в конфиге

я пытался их включать но резултат был тот же, можете показать как и где их включать, может просто я опять тупанул

**Reply by numbereleven-a** [reply]
> > У вас выключены все дурилки в конфиге
> 
> я пытался их включать но резултат был тот же, можете показать как и где их включать, может просто я опять тупанул

NFQWS_ENABLE=

**Reply by 8hrsk** [reply]
К тому же, желательно использовать скрипт 50-discord, а так же дополнительные udp порты для войсов. Подробнее инфу можно найти в этом же разделе. Есть небольшой гайдик по настройке запрет под войсы
